trunk/SUAVE/Methods/Utilities/latin_hypercube_sampling.py

```diff
@@ -0,0 +1,94 @@
+# latin_hypercube_sampling.py
+#
+# Created:  Jul 2016, R. Fenrich (outside of SUAVE code)
+# Modified: Mar 2017, T. MacDonald
+
+import numpy as np
+
+## If needed for mapping to normal distribution:
+#from scipy.stats.distributions import norm 
+
+def latin_hypercube_sampling(num_dimensions,num_samples,criterion=''):
+    
+    n = num_dimensions
+    samples = num_samples
+    
+    segsize = 1./samples
+    lhd = np.zeros((samples,n))
+    
+    if( criterion == "" ): # sample is randomly chosen from within segment
```

Comment:
I imagine every time a new set of design variable samples needs to be created. The initial case would be for CFD calculation points so comparatively trivial. In general relative to other script I think pretty rarely but I can try changing it anyway.